src/Storefront/Resources/app/storefront/webpack.config.js

```diff
@@ -57,7 +57,7 @@ let webpackConfig = {
                 hot: true,
                 compress: false,
                 disableHostCheck: true,
-                port: 9999,
+                port: parseInt(process.env.STOREFRONT_ASSETS_PORT || 9999),
```

Comment:
Add radix argument to `parseInt`, since it is in line with Shopware's usage of the function and considered good practice.

```suggestion
                port: parseInt(process.env.STOREFRONT_ASSETS_PORT || 9999, 10),
```

Reply:
I used the implementation in `src/Storefront/Resources/app/storefront/webpack.config.js:28` as inspiration, which says:

```php
const proxyPort = parseInt(process.env.STOREFRONT_PROXY_PORT || 9998);
```

How should I proceed?

Reply:
@lukewenkers @rherwig Is there any suggestion on what I should do?

Reply:
It would be nice if you can implement the changes of @rherwig. The additional argument is considered best practise and is seen as an additional safety.

Reply:
I've added the radix argument ☺️